src/gui/EntryPreviewWidget.cpp

```diff
@@ -109,6 +113,16 @@ EntryPreviewWidget::EntryPreviewWidget(QWidget* parent)
 
 EntryPreviewWidget::~EntryPreviewWidget() = default;
 
+bool EntryPreviewWidget::eventFilter(QObject* object, QEvent* event)
+{
+    if (object == m_ui->entryTotpLabel && event->type() == QEvent::MouseButtonDblClick) {
+        clipboard()->setText(m_ui->entryTotpLabel->text().remove(" "));
```

Comment:
I don't like this. There is always the risk of reading in rubbish when you read back stuff from UI elements (famous KDE example where they would randomly add & accelerators everywhere). I suppose the risk is manageable here, but I'd prefer you use the original data source.

Reply:
Also, what's going on with the Linux CI?

Reply:
Good point, will fix to use entry data directly.